modules/apps/object/object-api/src/main/java/com/liferay/object/util/ObjectActionDataConverterUtil.java

```diff
@@ -15,17 +15,58 @@
 package com.liferay.object.util;
 
 import com.liferay.object.model.ObjectDefinition;
+import com.liferay.portal.kernel.json.JSONFactoryUtil;
 import com.liferay.portal.kernel.json.JSONObject;
+import com.liferay.portal.kernel.util.GetterUtil;
+import com.liferay.portal.kernel.util.UnicodeProperties;
+import com.liferay.portal.kernel.util.UnicodePropertiesBuilder;
 import com.liferay.portal.vulcan.dto.converter.DTOConverter;
 import com.liferay.portal.vulcan.dto.converter.DTOConverterRegistry;
 
+import java.util.ArrayList;
+import java.util.HashMap;
 import java.util.Map;
+import java.util.Objects;
 
 /**
  * @author Carolina Barbosa
  */
 public class ObjectActionDataConverterUtil {
 
+	public static UnicodeProperties convertParameters(
```

Comment:
Do you think it's necessary even with the class called `ObjectActionDataConverterUtil`?

Reply:
I understand your question. Perhaps my head was in the wrong place, but when I saw the method I immediately associated parameters to request parameters. So my suggestion was to prevent other developers to assume the same thing. What do you think? If you think that adding ObjectAction before the Parameters wouldn't help get rid of this issue, I'm okay with leaving as is.

Reply:
Got it! Thanks for explaining. I will make this change.